plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -268,22 +260,18 @@ func (l *persistentVolumeLabel) getGCECloudProvider() (*gce.Cloud, error) {
 		if err != nil || cloudProvider == nil {
 			return nil, err
 		}
-		gceCloudProvider, ok := cloudProvider.(*gce.Cloud)
-		if !ok {
-			// GetCloudProvider has gone very wrong
-			return nil, fmt.Errorf("error retrieving GCE cloud provider")
-		}
-		l.gceCloudProvider = gceCloudProvider
+		l.gcePVLabeler = cloudProvider.(cloudprovider.PVLabeler)
```

Comment:
do a conditional interface assertion so we can return an error instead of panicking?